internal/ethapi/api.go

```diff
@@ -2377,12 +2382,10 @@ func (s *BundleAPI) CallBundle(ctx context.Context, args CallBundleArgs) (map[st
 			jsonResult["error"] = result.Err.Error()
 			revert := result.Revert()
 			if len(revert) > 0 {
-				jsonResult["revert"] = string(revert)
+				jsonResult["revert"] = "0x" + common.Bytes2Hex(revert)
```

Comment:
Wouldn't hexutil.Encode fit better here?

Reply:
Yep you're right, I replaced `common.Bytes2Hex` by `hexutil.Encode` where appropriate